wsidicom/instance.py

```diff
@@ -543,8 +542,124 @@ def slice_thickness(self) -> float:
         """Return slice thickness."""
         return self._slice_thickness
 
+    def set_dataset_as_tiled_full(
+        self,
+        image_data: List[Tuple[Tuple[str, float], 'ImageData']]
+    ):
+        """Set dataset as a tiled full dataset. Make a new Shared functional
```

Comment:
Could you clarify in the docstring what is happening here, are you converting a TILED_SPARSE to TILED_FULL, are setting the metadata to TILED_FULL because the image data is already tiled full. The wording 'Set dataset as a tiled full dataset" is a little bit unspecific.